services/venice-router/src/main/java/com/linkedin/venice/router/throttle/ReadRequestThrottler.java

```diff
@@ -86,11 +78,8 @@ public ReadRequestThrottler(
         routingDataRepository,
         routerConfig.getMaxReadCapacityCu(),
         stats,
-        routerConfig.getPerStorageNodeReadQuotaBuffer(),
```

Comment:
Can we get rid of this config: `getPerStorageNodeReadQuotaBuffer` from VeniceRouterConfig as well? Also the corresponding config entries in `ConfigKeys`?